### ISSUE TITLE
Vertical alignment/height of some embedded components is wrong

### ISSUE THREAD

**Issue by nkohari** [auth0/cosmos#625]
We discussed this in #575. At some point a bug was introduced involving the vertical alignment/height of some embedded components. Here are some examples:

![](https://user-images.githubusercontent.com/1576/41159686-5f901a8c-6afb-11e8-8fe3-372d37e22782.png)

![](https://user-images.githubusercontent.com/1576/41159710-770d69ee-6afb-11e8-97cb-c78698dd619a.png)

![](https://user-images.githubusercontent.com/1576/41159795-ae9aa67e-6afb-11e8-8102-e9a7b0696c10.png)

It seems like it might just be an issue with `line-height`?

**Comment by landitus** [comment]
I'm interested to see if this is related to the resets or the global styles for line-height. Since this was not happening before #575, do we know what caused this?

**Comment by siddharthkp** [comment]
I agree that this seems like a case of inconsistent resets. I'll try to find the root cause.

It became visible outside create-react-app = #575 (don't kill the messenger 😅)

**Comment by landitus** [comment]
thanks @siddharthkp !!!
